static/script.js

```diff
@@ -0,0 +1,29 @@
+function toggleMenu() {
+    const menu = document.getElementById('menu');
+    const overlay = document.getElementById('overlay');
+    const isOpen = menu.classList.contains('open');
+
+    if (isOpen) {
+        menu.classList.remove('open');
+        overlay.classList.remove('active');
+    } else {
+        menu.classList.add('open');
+        overlay.classList.add('active');
+    }
+}
+
+// Optional: Close menu when clicking outside
+document.addEventListener('click', function (event) {
```

Comment:
better only add this event when the menu is opened

Reply:
Done - I did note get the menu animation suppressed, but anyways